templates/logging-sidecar-configmap.yaml

```diff
@@ -32,60 +32,12 @@ data:
           component: "${COMPONENT:--}"
           workspace: "${WORKSPACE:--}"
           release: "${RELEASE:--}"
-
-      filter_common_logs:
-        type: filter
-        inputs:
-          - transform_syslog
-        condition:
-          type: "vrl"
-          source: '!includes(["worker","scheduler"], .component)'
-
-      filter_scheduler_logs:
-        type: filter
-        inputs:
-          - transform_syslog
-        condition:
-          type: "vrl"
-          source: 'includes(["scheduler"], .component)'
-
-      filter_worker_logs:
-        type: filter
-        inputs:
-          - transform_syslog
-        condition:
-          type: "vrl"
-          source: 'includes(["worker"], .component)'
-
-      transform_task_log:
-        type: remap
-        inputs:
-          - filter_worker_logs
-          - filter_scheduler_logs
-        source: |-
-          # Parse Syslog input. The "!" means that the script should abort on error.
-          . = parse_json!(.message)
```

Comment:
This was broken and errored silently every time

Reply:
The component wise transforms was to separate out worker and scheduler logs from the message payload